README.md

```diff
@@ -70,6 +70,8 @@ please read the file [build_system.md](documentation/build_system.md).
 | [HiKey Board (HiSilicon Kirin 620)](https://www.96boards.org/products/hikey/)|`PLATFORM=hikey`|
 | MediaTek MT8173 EVB Board|`PLATFORM=mediatek-mt8173`|
 | Texas Instruments DRA7xx|`PLATFORM=ti-dra7xx`|
+| [FSL ls1021a](http://www.freescale.com/products/arm-processors/qoriq-arm-processors:QORIQ-ARM?cof=0&am=0)|`PLATFORM=ls-ls1021atwr`|
+| [FSL ls1021a](http://www.freescale.com/products/arm-processors/qoriq-arm-processors:QORIQ-ARM?cof=0&am=0)|`PLATFORM=ls-ls1021aqds`|
```

Comment:
PLATFORM=ls-ls1021aqds is an eval. board and not a customer board, so I will remove this and keep only PLATFORM=ls-ls1021atwr.